### ISSUE TITLE
"request entity too large" when editing/closing challenges

### ISSUE THREAD

**Issue by Alys** [HabitRPG/habitica#7615]
When performing actions on popular challenges, you will sometimes see the error "**request entity too large**" and the action you tried to take will fail. This used to occur often (https://github.com/HabitRPG/habitrpg/issues/1471) but will be less common now that we're on API v3. However I want to track it, and maybe we'll find ways to mitigate it further.

I just noticed it on the challenge [Pushup Challenge](https://habitica.com/#/options/groups/challenges/50c9d584-3a8b-48dc-8941-f84b47375d92) when I tried to edit it to remove a swear word. The challenge itself is small (only a few tasks), but I am assuming there are very many participants - I'll check when I have more time. No participants at all are shown on the challenge page.

[edited to remove mention of guilds - see comment below]

**Comment by paglias** [comment]
@Alys the problem is probably that all the members data is sent to the server. It should only affect challenges as for groups we fixed the problem

**Comment by Moocie** [comment]
**Breathe Real Air (Air-Pollution prevention challenge)**
Hello! I wanted to add two tasks to help prepare for cyclones and prevent heat strokes, but it says request entry too large. There are only 49 people using the challenge. Tavern challenge and an old one. https://habitica.com/#/options/groups/challenges/d3ba70c1-12c5-42b0-bb1a-99f4402d8ba3 
I was able to edit out a word and save, but was unable to add the tasks no matter how small I made them (even removing the emojis). 
I even tried adding a word and saving but that didn't work.

New update: Apparently The tasks have been added. It doesn't seem to be because I added it in small parts because I tried several times again and it didn't work.
